babel.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = api => {
 
 	return {
 		ignore: ['node_modules', '**/__snapshots__/**', '**/__tests__/**'],
+		plugins: ['babel-plugin-dev-expression'],
```

Comment:
One thing about `babel-plugin-dev-expression` is that it transforms to a check against `process.env.NODE_ENV`, right? Does this mean that we have an implicit dependency on some other plugin (maybe "babel-plugin-transform-node-env-inline") to provide a `process.env` stub in the browser environment?

Reply:
Yeah I actually just remembered that after I left work last night. I'm gonna look into what the best solution is.

Reply:
I ended up just adding `babel-plugin-transform-node-env-inline`. I'm open ears for other ideas, but this seems the most straight forward. I really like the idea of trying to use `warning()` within the clay library, I think we can really benefit in creating helpful warnings.